src/promptflow/promptflow/_sdk/_serving/flow_invoker.py

```diff
@@ -130,13 +131,23 @@ def _init_executor(self):
             storage = DefaultRunStorage(base_dir=self._dump_to, sub_dir=Path(".promptflow/intermediate"))
         else:
             storage = self.storage
-        self.executor = FlowExecutor._create_from_flow(
-            flow=self._executable_flow,
-            working_dir=self.flow_entity.code,
-            connections=self.connections,
-            raise_ex=self.raise_ex,
-            storage=storage,
-        )
+        if isinstance(self.flow_entity, EagerFlow):
```

Comment:
It may not be a good practice to have if else here, can we use FlowExecutor.create for both scenario? Can we avoid the dependency of awaring EagerFlow/Flow here?

Reply:
further, do you guys think it would be a good idea to use `ExecutorProxy` instead of `Executor` here? So that we may complete cross-language support naturally

Reply:
Updated to use `FlowExecutor.create`

Reply:
ignore me. as offline synced, this should be part of python core, so `ExecutorProxy` shouldn't appear here